tmp/differences.json

```diff
@@ -0,0 +1,105 @@
+{
+    "common_names": [
+        "otelcol_query_latency",
+        "otelcol_query_responses",
+        "otelcol_query_requests"
+
+    ],
+    "v1_only_names": [
```

Comment:
names is not the only thing we are looking for, diff needs to be against a combination of name+labels

Reply:
Can you please specify which label is exactly needed?

Reply:
Do you mean help and type?

Reply:
When you have a metric like this:
`otelcol_processor_batch_metadata_cardinality{processor="batch",service_instance_id="40307288-f861-429b-90a3-b5b9a04110a3",service_name="jaeger",service_version="..."} 1`

`otelcol_processor_batch_metadata_cardinality` is the metric name. `processor`, `service_instance_id`, etc. are labels. The time series _identity_ is a combination of name and labels.

Reply:
Noted, thanks will add that as well

Reply:
Can you PTAL now, I have added the labels to the names if the metric has a label tag associated with it.